@here/harp-mapview/lib/BackgroundDataSource.ts

```diff
@@ -55,19 +52,23 @@ export class BackgroundDataSource extends DataSource {
         return tile;
     }
 
-    // Returns true if the tiling scheme changed, false otherwise.
-    private setReferenceDataSource(referenceDataSource: DataSource | undefined): boolean {
-        this.m_referenceDataSource = referenceDataSource;
-        let newTilingScheme = BackgroundDataSource.DEFAULT_TILING_SCHEME;
+    private setReferenceDataSource(referenceDataSource: DataSource | undefined): void {
```

Comment:
This complexity is actually not needed.
Just hardcode webMercatorTilingScheme for the background data source.
If it is really needed that the background data source needs to have a different tiling scheme we could make it explicit in the MapView options

Reply:
Yep, we had discussion with Andres about that. Will do as a quick fix, but for future I think the whole BackgroundDataSource should be removed and it's functionality integrated into OmvDataSource or TileDataSource.